LedgerHQ/ledger-live-desktop
# LIVE-1192 Fix app sizes in Manager

## 🦒 Context (issues, jira)
https://ledgerhq.atlassian.net/browse/LIVE-1192

<img width="1196" alt="Screenshot 2022-02-15 at 12 47 02" src="https://user-images.githubusercontent.com/4631227/154057387-aa2b5bfe-cb6d-45e8-a6cb-af31cf27695e.png">


## 💻  Description / Demo (image or video)
The latest version of the Ledger Nano S Plus allows for block sizes that go below 1kB. For apps that are below that size product wants us to display the size in kB with one decimal point. On that line, we changed the units to be `kB` and `MB` because it seems those are the recommended units.

## 🖤  Expectations to reach

PR must pass CI, merge develop if conflicts, do not force push. Thanks!

- **on QA**: at least one of these two checkboxes must be checked:
- **on delivery**: at least one of these two checkboxes must be checked: 

PR must pass CI, merge develop if conflicts, do not force push. Thanks!



## How to test this
- Having a Ledger Nano S Plus on the latest greatest version
- Use provider 12
- Enter the manager and check that the sizes below one kB are shown as `0.X kB` and not in bytes.
